deb/build/debian/control

```diff
@@ -2,7 +2,7 @@ Source: @@ARTIFACTNAME@@
 Section: devel
 Priority: extra
 Maintainer: @@AUTHOR@@
-Build-Depends: debhelper (>= 9)
+Build-Depends: debhelper (>= 9), dh-systemd
```

Comment:
I have a feeling this may break Ubuntu 14.04?

Reply:
I think there should be a branch created just for trusty.

Though trusty supports systemd, it's not installed by default and is buggy.